src/commands/plugins.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(clippy::almost_swapped)] // needed for clap derive
```

Comment:
Should we link to the [issue](https://github.com/clap-rs/clap/issues/4857) where this is being fixed? That makes it easy to tell when this should be safe to remove.